app/ios/Plugin/AppPlugin.swift

```diff
@@ -0,0 +1,87 @@
+import Foundation
+import Capacitor
+
+@objc(AppPlugin)
+public class AppPlugin: CAPPlugin {
```

Comment:
This should use a deinit method to unregister from NotificationCenter so it won't crash if it gets released (likely under embedded use).